felix/daemon/daemon.go

```diff
@@ -57,7 +57,6 @@ import (
 	"github.com/projectcalico/calico/felix/buildinfo"
 	"github.com/projectcalico/calico/felix/calc"
 	"github.com/projectcalico/calico/felix/config"
-	_ "github.com/projectcalico/calico/felix/config"
```

Comment:
I'm not sure why this has an underscore import in the first place..

Reply:
No idea